modules/capacity_factors_wind_pv/workflow/rules/downloads.smk

```diff
@@ -0,0 +1,45 @@
+# We recommend adding rules that download necessary files here.
+
+
+rule download_units:
+    message:
+        "Download spatial zones."
+    params:
+        url=config["resources"]["spatial_units"],
+    output:
+        "resources/units_{resolution}.geojson",
+    conda:
+        "../envs/shell.yaml"
+    localrule: True
+    shell:
+        "curl -sSLo {output} '{params.url}'"
```

Comment:
Please add a check to deactivate this (see https://github.com/calliope-project/ec_modules/pull/39)